store/registration_test.js

```diff
@@ -0,0 +1,36 @@
+let user = {
+    firstName: 'Anton',
+    state: 'Florida',
+    lastName: 'Kozyrev',
+    address: '2701 Sandy Ford',
+    city: 'Ford',
+    zipCode: '32011',
+    mobilePhone: '+169852147375',
+    password: '987456',
+}
+
+Feature('store');
+
+Before(({ I }) => {
+    I.openStore();
+});
+
+Scenario('auth', async ({ I, homePage, authPage, createAccountPage, myAccountPage }) => {
+    I.openStore();
+    homePage.clickSignIn();
+    authPage.fillRegistrationEmail(await I.getRandomEmail());
+    authPage.clickCreateAccount();
+    createAccountPage.submitNewUserFields(user);
+    myAccountPage.verifyMyAccountPage();
+}).tag('auth');
+
+Scenario('custom helper', async ({ I, tryToHelper }) => {
+    I.amOnPage('http://automationpractice.com/index.php?id_product=3&controller=product#/size-s/color-orange');
+    let result = await tryToHelper.checkElementIsVisible({ id: 'add_to_cart' });
```

Comment:
move `tryTo()` to `buyProduct_test.js`.
if page has price, test should do all stuff(grab price, buy product).
if no price - it should proceed to next product from Data Table.